toolkit/docs/building/building.md

```diff
@@ -809,6 +809,7 @@ To reproduce an ISO build, run the same make invocation as before, but set:
 | Variable                      | Default                                                                                                | Description
 |:------------------------------|:-------------------------------------------------------------------------------------------------------|:---
 | LOG_LEVEL                     | info                                                                                                   | Console log level for go tools (`panic, fatal, error, warn, info, debug, trace`)
+| LOG_COLOR                     | auto                                                                                                   | Console log color for go tools (`always, auto, never`). `always` enables color in both logs and terminal output, `auto`(`default` option) enables color in terminal output, and `never` disables color in all.
```

Comment:
Question: does the code now indeed enables the colouring in logs for the `always` option?

Reply:
Yes, this is how the logs look in vscode and in terminal when we 'cat' them, with LOG_COLOR=always
![image](https://github.com/microsoft/CBL-Mariner/assets/58672330/9d689cfb-c956-4726-ada0-6b345d2712f3)